superset-frontend/src/components/DeleteModal/index.tsx

```diff
@@ -80,10 +80,11 @@ export default function DeleteModal({
       disablePrimaryButton={disableChange}
       onHide={hide}
       onHandledPrimaryAction={confirm}
-      primaryButtonName={t('delete')}
+      primaryButtonName={t('Delete')}
```

Comment:
Bycatch, another mis-capitalized button. Maybe @michael-s-molina has some regex-y grep magic to see if there are more of these :D